common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -78,11 +78,16 @@ def library_pr_name
               "#{from_version_msg(old_library_requirement(dependencies.first))}" \
               "to #{new_library_requirement(dependencies.first)}"
           else
-            names = dependencies.map(&:name)
-            "requirements for #{names[0..-2].join(', ')} and #{names[-1]}"
+            names = dependencies.map(&:name).uniq
+            if names.count == 1
+              "requirements for #{names.first}"
+            else
+              "requirements for #{names[0..-2].join(', ')} and #{names[-1]}"
+            end
```

Comment:
💡 Since we have ActiveSupport available, we could use `#to_sentence` for this case ([docs](https://guides.rubyonrails.org/active_support_core_extensions.html#to-sentence))

Reply:
I had this change stashed so I just pushed it up here in case you want to use any of it: https://github.com/dependabot/dependabot-core/commit/ce8eb68a4ad41dd4940d68f79dc4123e0866813b

Reply:
Thanks for the suggestions! `#to_sentence` looks ideal but because it requires adding some new active_support imports I think it'd be better to make that change as a separate PR focused on adding active_support to simplify this class.